pkg/service/repair/controller_test.go

```diff
@@ -47,125 +47,81 @@ func (s controllerTestSuite) newRowLevelRepairController() *rowLevelRepairContro
 func TestRowLevelRepairController(t *testing.T) {
 	ctx := context.Background()
 
-	t.Run("TryBlock", func(t *testing.T) {
+	t.Run("TryBlock with maxIntensity to follow the to ensure number of token ranges per job", func(t *testing.T) {
 		s := makeControllerTestSuite()
 		ctl := s.newRowLevelRepairController()
 
-		for i := 0; i < controllerTestDefaultRangesLimit; i++ {
-			if ok, _ := ctl.TryBlock([]string{"a", "b", "c"}); !ok {
-				t.Fatal("TryBlock() failed to block")
-			}
-		}
-		if ok, _ := ctl.TryBlock([]string{"a", "b", "c"}); ok {
-			t.Fatal("TryBlock() unexpected success")
-		}
-		ctl.Unblock(allowance{
-			Replicas: []string{"a", "b", "c"},
-			Ranges:   1,
-		})
-		if ok, _ := ctl.TryBlock([]string{"a", "b", "c"}); !ok {
+		const intensity = 1000
+
+		s.intensityHandler.SetIntensity(ctx, intensity)
+
+		ok, a := ctl.TryBlock([]string{"a", "b", "c"})
+		if !ok {
 			t.Fatal("TryBlock() failed to block")
 		}
+		if a.Ranges != intensity {
```

Comment:
Should be ">" - exactly as the error message below states.
Repair task is allowed to pick less than "intensity" ranges, e.g. in the last hunk.